requirements.txt

```
@@ -1 +1,2 @@
+ansible-pylibssh
```

Can we set a min version to avoid the reboot traceback? (whenever it releases, anyway)

```suggestion
ansible-pylibssh >= 0.2.0
```